Tests/OpenCombineTests/AnySubscriberTests.swift

```diff
@@ -138,15 +138,22 @@ final class AnySubscriberTests: XCTestCase {
 
         let expectedEvents: [TrackingSubject<Int>.Event] =
             [.subscription("Subject")] + events.compactMap(subscriberEventToSubjectEvent)
+                .throughFirstCompletion()
 
         XCTAssertEqual(subject.history, expectedEvents)
 
         let shuffledEvents = events.shuffled()
 
         publishEvents(shuffledEvents, erased)
 
-        let expectedShuffledEvents =
-            shuffledEvents.compactMap(subscriberEventToSubjectEvent)
+//        let expectedShuffledEvents =
+//            shuffledEvents.compactMap(subscriberEventToSubjectEvent)
+        // TODO: Because Apple changed the behavior of AnySubscriber to become inactive
+        // after it receives its first `.completion`, this shuffled testing no longer
+        // really makes any sense.
+        // Depending on the original goal of the shuffled testing, a new test could be
+        // written.
+        let expectedShuffledEvents = [TrackingSubject<Int>.Event]()
```

Comment:
The original goal was to make sure that `AnySubscriber` doesn't do anything smart and just passes the events to the underlying subscriber as-is. Since this is no longer the case, we can indeed remove this whole thing.